pkg/build/message.go

```diff
@@ -0,0 +1,19 @@
+package build
+
+import (
+	"fmt"
+)
+
+const (
+	messageScriptName = "message.sh"
+	messageScriptsDir = "message"
+)
+
+func configureMessage(b *Builder) error {
```

Comment:
I'd suggest to convert this to a method instead. It's not in the same file as the `Builder` definition but it is allowed.